index.html

```diff
@@ -12,88 +12,196 @@
+            <li>
+              <a href=""><img src="Asset/vector1git.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/LinkidIn.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/angellist.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/Twitter.png" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="Asset/medium.png" alt="" /></a>
+            </li>
+          </ul>
         </div>
-      </section>
-      <div class="socialMediaGroup">
-        <ul class="social">
-          <li>
-            <a href=""><img src="Asset/vector1git.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="Asset/LinkidIn.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="Asset/angellist.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="Asset/Twitter.png" alt="" /></a>
-          </li>
-          <li>
-            <a href=""><img src="Asset/medium.png" alt="" /></a>
-          </li>
-        </ul>
-      </div>
-      <div class="MyResume">
-        <button>Get My Resume</button>
-      </div>
-    </div>
-    <div class="accordion">
-      <div class="accordion-header">
-        <h3>Language</h3>
-        <i class="icon"><img src="Asset/icon.png" alt="" /></i>
-      </div>
-      <div class="accordion-content">
-        <div class="ListProgramming1">
-          <img
-            src="Asset/JS.png"
-            alt="JavaScript Image"
-            class="Javascript-image"
-          />
-          <p class="textJavascript">Javascript</p>
+        <div class="MyResume">
+          <button>Get My Resume</button>
         </div>
-        <div class="ListProgramming2">
-          <img
-            src="Asset/HTML5.png"
-            alt="JavaScript Image"
-            class="Javascript-image"
-          />
-          <p class="textJavascript">HTML</p>
+      </div>
+      <div class="accordion">
+        <div class="accordion-header">
+          <h3>Language</h3>
+          <i class="icon"><img src="Asset/icon.png" alt="" /></i>
         </div>
-        <div class="ListProgramming3">
-          <img src="Asset/CSS3.png" alt="JavaScript Image" class="CSS-image" />
-          <p class="textJavascript">CSS</p>
+        <div class="accordion-content">
+          <div class="ListProgramming1">
+            <img
+              src="Asset/JS.png"
+              alt="JavaScript Image"
+              class="Javascript-image"
+            />
+            <p class="textJavascript">Javascript</p>
+          </div>
+          <div class="ListProgramming2">
+            <img
+              src="Asset/HTML5.png"
+              alt="JavaScript Image"
+              class="Javascript-image"
+            />
+            <p class="textJavascript">HTML</p>
+          </div>
+          <div class="ListProgramming3">
```

Comment:
- [ ] Please enclose the technology categories ("Languages", "Frameworks" and "Skills") in a `ul`, each of them in a `li` and ...
- [ ] also the technology in each category(e.g., "JavasScript", "HTML" and "CSS") in a list `ul` and each of them a `li`

Example snippet:


```html

 <ul class="skill-box">
       <li>
         <h3>Languages</h3>
            <ul>
                <li>JavaScript</li>
                 <li>HTML</li>
                 <li>CSS</li>
            </ul>
        </li>
        <li><h3>Frameworks</h3></li>
        <li><h3>Skills</h3></li>
 </ul>


```